packages/gatsby-theme-blog-core/gatsby-config.js

```diff
@@ -2,7 +2,7 @@ const withDefaults = require(`./utils/default-options`)
 
 module.exports = themeOptions => {
   const options = withDefaults(themeOptions)
-  const { mdx = true } = themeOptions
+  const { mdxOtherwiseConfigured = false } = themeOptions
```

Comment:
I really like the improved clarity of this option name. Can we keep a check in here for the current `mdx` option to prevent this from being a breaking change? (e.g. `mdx !== false`)

Reply:
Good call. Though clarifying question, aren't themes a snapshot in time? Do we have a process for updating the theme you're using?

Reply:
Since themes are (usually) installed from npm, they can be updated just like any other package or plugin. This isn't a problem with *starters,* but with themes we have to be mindful of breaking changes

Reply:
Ah! That makes sense. Not sure why I flipped that around in my head.